rancher/dashboard
# 8261 broken component events link

Fixes #8261 

- fix issue with formatter for events links that breaks the ui

## To test
- Provision a Digital Ocean `rke2` cluster
- once provisioned, go to the DO `rke2` cluster page -> `nodes`
- `Cordon` the node
- `Uncordon` the node (this will generate node related events)
- Go to the `cluster explorer` page for this cluster and make sure UI doesn't break
- Go to the `events` list view for this cluster and make sure UI doesn't break
